static/app/components/button.tsx

```diff
@@ -193,6 +195,16 @@ type LinkButtonProps =
   | HrefLinkButtonPropsWithChildren
   | HrefLinkButtonPropsWithAriaLabel;
 
+/**
+ * Default sizes to use for SVGIcon
+ */
+const ICON_SIZES = {
```

Comment:
can we shove this button-specific stuff into `function IconDefaultsProvider() {...}` instead of expanding button with it?

Reply:
and if we can avoid copy+paste of the list that would be better